src/PerformanceTests/Common/Statistics/SimpleStatisticsFeature.V6.cs

```diff
@@ -22,6 +22,8 @@ public SimpleStatisticsFeature()
 
         protected override void Setup(FeatureConfigurationContext context)
         {
+            Console.WriteLine("setup SimpleStatisticsFeature");
```

Comment:
@timbussmann let's remove this?

Reply:
Not sure why this was added? I think these write lines can be removed.

Reply:
some debugging leftovers